Mathmagician/Program.cs

```diff
@@ -10,6 +10,62 @@ class Program
     {
         static void Main(string[] args)
         {
+            string requestInput = "What would you like me to do?";
+            string requestNumber = "How many should I print?";
+            string prompt = "> ";
+            string makeIntoAnInt = "";
+            int printThisManyNumbers = 0; 
+            Console.WriteLine("You have summoned the Mathmagician! You must be in need of numbers.");
+            Console.WriteLine("VERY WELL! Input one of my magic words, press the Enter key and watch in awe.");
+            Console.WriteLine("The magic words are: integer, fibonacci, prime, even and odd.");
+            Console.WriteLine(requestInput);
+
+            string userInput = Console.ReadLine().ToLower();
+            if (userInput == "integer")
+            {
+                Console.WriteLine("AHA! INTEGERS! " + requestNumber);
+                makeIntoAnInt = Console.ReadLine();
+                printThisManyNumbers = Int32.Parse(makeIntoAnInt);
```

Comment:
Use you use TryParse, because if your input value can't be made into an int, it's going to throw an exception. TryParse allows the parsing to fail gracefully. For an example, look at the unit testing repo I pasted in slack (in the Program.cs)